roles/post_install/openstack_tasks/tasks/flavors.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Note! - The task splitted into two different tasks because
```

Comment:
What are the two different tasks that this is split into?

It appears that there is only one item defined in `flavors` in `roles/post_install/openstack_tasks/defaults/main.yml`

Please remove this comment if it is not relevant/accurate.

Reply:
Ack. Will do.